NeuralNetwork.py

```diff
@@ -0,0 +1,118 @@
+import random
+from scipy.special import expit # fast logistic function
+
+
+class NeuralNetwork:
+
+    def __init__(self, size):
+        self.LEARNING_RATE = 0.5
+        self.layers = []
+        self.init_layers(size)
+        self.init_weights(size)
+
+    def init_layers(self, size):
+        for i in range(1, len(size)):
+            self.layers.append(NeuronLayer(size[i]))
+
+    def init_weights(self, size):
+        for j in range(1, len(size)):
+            for h in range(size[j]):
+                for i in range(size[j-1]):
+                    self.layers[j-1].neurons[h].weights.append(random.random())
+
+    def feed_forward(self, inputs):
+        output = inputs
+        for i in range(len(self.layers)):
```

Comment:
Instead of this function body you could write one line list comprehension.

Reply:
I don't know how to do it.